pkg/kopia/utils.go

```diff
@@ -32,6 +30,10 @@ import (
 	"github.com/pkg/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	"path/filepath"
+	"strings"
```

Comment:
I don't think they are grouped together correctly.

Reply:
Will correct it !